saleor/settings.py

```diff
@@ -534,6 +534,10 @@ def get_host():
         "task": "saleor.giftcard.tasks.deactivate_expired_cards_task",
         "schedule": crontab(hour=0, minute=0),
     },
+    "update-stocks-quantity-allocated": {
+        "task": "saleor.warehouse.tasks.update_stocks_quantity_allocated_task",
+        "schedule": timedelta(days=1),
```

Comment:
I would change it to `crontab` for some night hour when potentially there is smaller traffic.

Reply:
Why do we need this at all? Migration isn't enough?

Reply:
We want to keep it to check for any inconsistency in logs for a while, however running at night is a good idea, changing to crontab.